### PR TITLE
BLD-17 Add google analytics code

### PR DESCRIPTION
**Pull request by radman**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,10 +6,10 @@
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+
+  <%= render 'layouts/google_analytics' if Rails.env.production? %>
```
Comment (leoping) [inline app/views/layouts/application.html.erb]: our Rails.env.production? will always be true between staging and production right?  Need to check the Herok Env variable instead?

Comment (radman) [inline app/views/layouts/application.html.erb]: yup, you're right

**Comment by radman** [comment]
Now using env variables to enable analytics @leoping

**Comment by leoping** [comment]
:+1:

**Pull request merged**